README.md

```diff
@@ -44,9 +43,9 @@ All trial projects can provision a free trial phone number for testing. Here's h
 ![Search Results](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-twilio/master/ballerina/resources/search-phone-number.png)
 > **Notice:** Many countries require identity documentation for Phone Number compliance. Requests to provision phone numbers with these regulations will be required to select or add the required documentation after clicking Buy in Console. To see which countries and phone number types are affected by these requirements, please see twilio's [Phone Number Regulations](https://www.twilio.com/guidelines/regulatory) site.
 
-### Step 3: Obtain a Twilio account SId with auth token.
+### Step 3: Obtain a Twilio account Sid with auth token
```

Comment:
Does it pronounced as `Sid` or `SId` ?

Reply:
It's pronounced as "Sid" not "SId." The term "SID" stands for ["String Identifier"](https://www.twilio.com/docs/glossary/what-is-a-sid) in the context of Twilio, and it's typically pronounced as a single word rather than as individual letters.

There is a related issue as well: https://github.com/ballerina-platform/ballerina-library/issues/4960